src/logger/HttpRules_Test.go

```diff
@@ -0,0 +1,959 @@
+package logger
+
+import (
+	"regexp"
+	"sync"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestChangesDefaultRules(t *testing.T) {
+	httpRules, _ := GetHttpRules()
```

Comment:
It seems like an error is getting returned here, do we really want to ignore it?